#46 Grafana dashboard

grighetto opened the pull request:
Report performance metrics from both NoSQLBench jobs to a standalone graphite/grafana/prometheus setup.

Resolves #46

grighetto commented on helm/adelphi/config/grafana/nb4-new-dashboard.json:
```diff
@@ -0,0 +1,1405 @@
+{
```
This is the default dashboard that comes with NB. We'll soon replace it with something that makes more sense for Adelphi, like some kind of side-by-side comparison with sensitive performance metrics. New ticket coming up for this.

grighetto replied:
https://github.com/datastax/adelphi/issues/91

jdonenine commented on helm/adelphi/templates/prometheus-configmap.yaml:
```diff
@@ -0,0 +1,39 @@
+        timeout: 10s
+
+    scrape_configs:
+      - job_name: prometheus
```
I don't really think it much matters, but do we need prometheus scraping itself in this scenario? Not sure it really adds any overhead or not, but I'd imagine our users would never look at the metrics produced by doing this.

grighetto replied:
agreed, I'll remove it, that's coming from the stock NoSQLBench config

grighetto replied:
Done

jdonenine reviewed (commented):
Looks good @grighetto everything ran as expected and could be seen in Grafana, this is a great add for usability 🎉 .

Just two minor things I noted.

We'll also want to update the docs with info about this feature as well - let's cover that as a separate issue though and go ahead and get this in.

grighetto commented on helm/adelphi/config/grafana/nb4-new-dashboard.json:
```diff
@@ -0,0 +1,1405 @@
+    ]
+  },
+  "timezone": "",
+  "title": "NB4 Dashboard",
```
Renamed to "Adelphi NoSQLBench" for now, we can change it again if needed.

grighetto commented on helm/adelphi/templates/grafana.yaml:
```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Pod
```
k8s service added for grafana

grighetto commented:
@jdonenine Thanks for the quick review, I addressed all your comments and tested on GKE.
If you don't have any other feedback, I'll go ahead and merge this.

jdonenine reviewed (approved):
Thanks for the changes. Good to go!
